data/campaigns/Eastern_Invasion/scenarios/07a_The_Crossing.cfg

```diff
@@ -365,6 +376,13 @@
                             message= _ "Grug say join you will he."
                         [/message]
 
+                        # TODO: Instead of a narrator messages, spawn the ogres on map next to Grug, maybe mention them in dialog.
+                        [message]
+                            image=wesnoth-icon.png
+                            speaker=narrator
+                            message= _ "Two ogres have been added to your recall list!"
+                        [/message]
+
```

Comment:
When opening new PRs, I'm going to drop this change. I feel it still needs work, and is better left unmerged until someone has a vision of exactly how to inform the player of it.